fuel-vm/src/interpreter/executors/opcodes_impl.rs

```diff
@@ -1637,7 +1637,45 @@ where
     ) -> IoResult<ExecuteState, S::DataError> {
         interpreter.gas_charge(interpreter.gas_costs().lb())?;
         let (a, b, imm) = self.unpack();
-        interpreter.load_byte(a, interpreter.registers[b], imm.into())?;
+        interpreter.load_u8(a, interpreter.registers[b], imm)?;
+        Ok(ExecuteState::Proceed)
+    }
+}
+
+impl<M, S, Tx, Ecal, V> Execute<M, S, Tx, Ecal, V> for fuel_asm::op::LQW
+where
+    M: Memory,
+    S: InterpreterStorage,
+    Tx: ExecutableTransaction,
+    Ecal: EcalHandler,
+    V: Verifier,
+{
+    fn execute(
+        self,
+        interpreter: &mut Interpreter<M, S, Tx, Ecal, V>,
+    ) -> IoResult<ExecuteState, S::DataError> {
+        interpreter.gas_charge(interpreter.gas_costs().lw())?;
```

Comment:
It's fine to use the same gas_costs (the same value ok, but talking about the exact same field), it can't cause any issue elsewhere to not have gas defined for this opcode ? (Just double checking)

Reply:
My understanding is yes, but maybe @xgreenx knows about some issues?